pkg/ocm/sca/sca_test.go

```diff
@@ -19,7 +19,7 @@ var (
 func Test_SCAController_SecretIsCreated(t *testing.T) {
 	kube := kubefake.NewSimpleClientset()
 	coreClient := kube.CoreV1()
-	scaController := New(context.TODO(), coreClient, nil, nil)
+	scaController := New(context.TODO(), coreClient, nil, nil, nil)
```

Comment:
nit (not related to this change): we could change `context.TODO` to `context.Background`

Reply:
Reverted, as the config is not needed anymore